src/data/reducers/shopify.js

```diff
@@ -23,26 +26,37 @@ const OPEN_CART = 'shopify/OPEN_CART';
 const CLOSE_CART = 'shopify/CLOSE_CART';
 const CART_COUNT = 'shopify/CART_COUNT';
 
+const CHECKOUT_ID_LOCAL_STORAGE_KEY = 'SHOPIFY_CHECKOUT_ID';
+const CHECKOUT_ID_FROM_LOCAL_STORAGE = window.localStorage.getItem(CHECKOUT_ID_LOCAL_STORAGE_KEY);
+
 const initialState = {
```

Comment:
Is it worth removing unnecessary states for now (add back later when needed) to avoid potential confusion amongst devs? I'm not strongly pushing for one or the other - just curious.

Reply:
I don't think anyone else will be working on this checkout stuff except you and I so we can probably leave it in there for now

Reply:
great question! I actually thought about this but forgot to leave a comment haha, here are my thoughts:

* I don't actually know what the other states do or if we'll need that information down at any point (hence why I added "unsure what this does" comments)
* I didn't remove them YET in case we needed it down the line
* But you probably know more than I do, so if we can remove them for sure let's remove them! I just left them in for now but also added a comment to note that they're not used 🤷  

I think that in general, we should delete code whenever possible — deleting code is almost always a good thing 🙃 . Feel free to delete in your own branches later on!